resources/views/home/_search_results.blade.php

```diff
@@ -25,7 +25,7 @@
 
             <a
                 class="search-result__more-button {{ $showMore ? '' : 'search-result__more-button--hidden' }}"
-                href="{{ route('search', ['mode' => $mode, 'query' => request('query')]) }}"
```

Comment:
I remember this being done intentionally to preserve the query as-is instead of changing `foo   ` to `foo` but I suppose it should be fine as long as the input box's text itself isn't changing 🤔